andela/casablanca-rc
# feat(theme): create new theme

#### What does this PR do?
- install bcrypt
- add styles for new theme
- add styles to email templates to reflect the new theme
- remove custom plugins from .gitignore
- remove unnecessary .gitkeep
#### Description of Task to be completed?
Users should see a new theme design
#### How should this be manually tested?
- clone the repo and change into the project directory
- start the application(instructions in the readme)
- visit the application in the browser 
- you should see the new theme design

#### What are the relevant pivotal tracker stories?
#157032076
#### Screenshots
![screencapture-localhost-3000-2018-05-04-10_39_07](https://user-images.githubusercontent.com/26048536/39621746-b00802f2-4f87-11e8-93e5-803d5a882a15.png)
![screencapture-localhost-3000-product-example-product-2018-05-04-10_40_03](https://user-images.githubusercontent.com/26048536/39621757-b730fc1e-4f87-11e8-9b59-ba9c5129b97c.png)
![screencapture-localhost-3000-2018-05-04-10_39_44](https://user-images.githubusercontent.com/26048536/39621722-9f89bee8-4f87-11e8-8750-db58a0145586.png)
